packages/core-data/src/queried-data/selectors.js

```diff
@@ -66,7 +66,10 @@ function getQueriedItemsUncached( state, query ) {
 
 			for ( let f = 0; f < fields.length; f++ ) {
 				const field = fields[ f ].split( '.' );
-				const value = get( item, field );
+				let value = { ...item };
```

Comment:
I don't think we need to clone the object here.

Reply:
My rationale was that if we directly assign `item` to `value` by reference, when we change `value` on L71, we'll also change `item`, which may eventually lead to unexpected results if we end up using `item` somewhere.

Or are you talking about the fact that `item` is not used after this potential mutation in this branch of the code, so we might as well mutate it?

Reply:
my rationale here is that "field" containers at least one element, so there's no way value is going to be item in the end, or am I wrong?

Reply:
I think you have a point. Removed in f00713e60d9a563b8b8349f81e394a33b0832c70